docs/faq/proof-of-stake/pos-general-faq.md

```diff
@@ -0,0 +1,123 @@
+
+The only cost for using this system are the fees you set. The txfee will be charged for the transaction of your ticket across the network. The ticketfee will be charged for incorporation into the PoS ticket pool, and is paid to the miners. If your ticket does not get mined, in case of a full buying window, the ticketfee will be returned to you along with the DCR paid for the ticket price.
+
+In case your ticket isn't selected to vote after 40960 blocks (about 4 months), the system will revoke your ticket and the DCR you paid for it will be transferred back into your wallet. You will not get the PoS reward. There is a low, about 2%, chance of this happening. The average time for a ticket to vote is 28 days.
+
+In case your ticket is selected to vote but your wallet cannot vote, e.g. it isn't online or unlocked, the system marks the ticket as missed and the DCR you paid for it will be transferred back into your wallet after the system revokes your ticket. You will not get the PoS reward. To avoid this scenario the use of a stake pool is advised.
+
+You will always get back the amount of DCR you paid for a ticket minus the applicable fees.
+
+---
+
+#### **7. What is Proof-of-Stake voting?**
+
+Because the block validation performed by the PoS system behaves like a voting system, it can be used to vote on other issues too.
+
+When selected to validate a block, the tickets vote on whether or not to approve the previously mined block. This requires 3 out of 5 selected tickets to vote yes.
+
+By adding another parameter to the ticket that doesn't interfere with the approval of the mined blocks, the system can track the amount of tickets using that parameter over a number of blocks. You can set this parameter in your wallet at any time before the ticket votes. 
+
+For example, you could choose the color of your ticket to be either red or blue, set that option, and the system will count for the next 1000 blocks how many red tickets and how many blue tickets there are. Maybe there are 3000 red tickets, 1500 blue tickets, and 500 that didn't select a colour.
+
+If you replace the red or blue option with a yes or no option, you have a voting system that can count votes over time. By using the PoS voting system anyone who holds DCR can vote on issues set in an agenda.
+
+This voting system can be used for any issue. For Decred the most prominent use will be to vote on hardforks.
+
+Currently, a platform for the community to submit and advocate a voting agenda is in development.
+
+---
+
+#### **8. What is hardfork voting?**
+
+Like any other cryptocurrency Decred might need to hardfork at some point.
+
+One of the agenda issues set for the PoS voting system could be a hardfork. If such an issue is set, the new version of the decred files will include the hardfork but it will not activate until the PoS system has voted on it.
```

Comment:
and a vote must succeed with a majority for the change to be locked in and then activated.